config/default/common/brand/about/welcome.md

```diff
@@ -0,0 +1,31 @@
+<h1>Welcome to Worldview<span class="version"> Version 3.16.0 (<a href="https://github.com/nasa-gibs/worldview/releases"
```

Comment:
As Minnie pointed out, this version shouldn't be hard-coded

Reply:
I think we crossed paths when you made this comment.   Addressed in https://github.com/nasa-gibs/worldview/pull/3798/commits/34e9153063ac73d63d221078c62c00ec79365b52